specification/hybridconnectivity/resource-manager/Microsoft.HybridConnectivity/PublicCloud/stable/2024-12-01/publicCloud.json

```diff
@@ -0,0 +1,1671 @@
+          },
+          {
+            "$ref": "#/parameters/Azure.ResourceManager.ResourceUriParameter"
+          },
+          {
+            "name": "solutionConfiguration",
+            "in": "path",
+            "description": "Represent Solution Configuration Resource.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-zA-Z0-9-]{3,63}$"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Azure operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/InventoryResourceListResult"
+            }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "../../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Inventory_ListByWidget": {
+            "$ref": "./examples/Inventory_ListByWidget.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/{resourceUri}/providers/Microsoft.HybridConnectivity/solutionConfigurations/{solutionConfiguration}/inventory/{inventoryId}": {
+      "get": {
+        "operationId": "Inventory_Get",
+        "tags": [
+          "Inventory"
+        ],
+        "description": "Get a InventoryResource",
+        "parameters": [
+          {
+            "$ref": "../../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/Azure.ResourceManager.ResourceUriParameter"
+          },
+          {
+            "name": "solutionConfiguration",
+            "in": "path",
+            "description": "Represent Solution Configuration Resource.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-zA-Z0-9-]{3,63}$"
+          },
+          {
+            "name": "inventoryId",
+            "in": "path",
+            "description": "Inventory resource",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Azure operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/InventoryResource"
+            }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "../../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Inventory_Get": {
+            "$ref": "./examples/Inventory_Get.json"
+          }
+        }
+      }
+    },
+    "/{resourceUri}/providers/Microsoft.HybridConnectivity/solutionConfigurations/{solutionConfiguration}/syncNow": {
```

Comment:
>syncNow [](http://example.com/codeflow?start=106&length=7)

may be more detailed about what this action is ? 

how about "syncWithSourceCloud" ?

Reply:
We do periodic sync of resources with source cloud, so we have an ondemand sync functionality so named it as syncNow

Reply:
@ramoka178, the POST API action name was approved by previous ARM review after discussion and it is already release for out customers. Chaning to new method name with break customers. Is there any new published guidance or this is just a suggestion?